sync-superchain.sh

```diff
@@ -50,8 +50,11 @@ process_network_dir() {
         chain_id=$(dasel -f "$toml_file" -r toml "chain_id" | tr -d '"')
         chain_name="$(basename "${toml_file%.*}")"
 
-        # Skip if chain_id is empty or Boba Sepolia (28882) or Boba Mainnet (288)
-        if [[ -z "$chain_id" || "$chain_id" -eq 28882 || "$chain_id" -eq 288 ]]; then
+        if [[ -z "$chain_id"
+              || "$chain_id" -eq 28882     # Boba Sepolia
+              || "$chain_id" -eq 288       # Boba Mainnet
+              || "$chain_id" -eq 42220 ]]; # Celo Mainnet
```

Comment:
Perhaps expand the comment to say why Celo is excluded? And the other chains if you know why.

Reply:
Yes good call. Will add an issue to the scr